bin/yauml

```diff
@@ -19,9 +19,10 @@
 #                  http://www.gnu.org/licenses/                                     *
 #************************************************************************************
 
-import sys, yaml, getopt, re, subprocess
+import os, sys, yaml, getopt, re, subprocess
 
-template_filename = '../templates/default.dot'
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+TEMPLATE_FILENAME = CURRENT_DIR + '/../templates/default.dot'
```

Comment:
I guess I missed the main explanation of you pull request which intends to introduce an "unconventionnal" installation.